protocol/v2/ssv/runner/attester.go

```diff
@@ -166,10 +160,15 @@ func (r *AttesterRunner) ProcessPostConsensus(logger *zap.Logger, signedMsg *spe
 		}
 		specSig := phase0.BLSSignature{}
 		copy(specSig[:], sig)
-
+		r.metrics.EndPostConsensus()
 		logger.Debug("🧩 reconstructed partial signatures",
-			zap.Uint64s("signers", getPostConsensusSigners(r.GetState(), root)))
+			zap.Uint64s("signers", getPostConsensusSigners(r.GetState(), root)),
+			fields.QuorumTime(r.metrics.GetPostConsensusTime()))
 
+		attestationSubmissionEnd := r.metrics.StartBeaconSubmission()
```

Comment:
this confused me at first, I think the var name should be or include "EndFunc"

Reply:
renamed